src/arcsf/data/data_utils.py

```diff
@@ -0,0 +1,244 @@
+import itertools
+
+import numpy as np
+from datasets import Dataset, load_dataset
+
+
+def get_indices_structured(
+    forgotten_author_numbers: list,
+    q_remove: int = 4,
+    n_authors: int = 200,
+    q_per_author: int = 20,
+):
+    """
+    Returns two flattened lists of Question--Answer indices given authors numbers to
+    remove/retain, in the case where the forget target can be easily structured.
+    Assumes equal questions per author, and sorted authors.
+
+        Parameters:
+                forgotten_author_numbers (list):    List of integers containing author
+                                                    indexes about which facts should
+                                                    be removed
+                n_authors (int):                    Total number of authors
+                q_removal (int):                    Number of questions/facts that
+                                                    should be removed
+                q_per_author (int):                 Integer defining the number of
+                                                    questions to remove per author
```

Comment:
Looking throughout I think this variable is always relating to the total number of questions per author in the TOFU dataset, not the number of questions to remove per author (which is `q_removal`)?